### PR TITLE
Added support for PHP 8 and composer 2

### PR DESCRIPTION
**Pull request by IgorKorytin**
### Added

- Support PHP `8.x`

### Changed

- Composer `2.x` is supported now

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I wrote unit tests for my code _(if tests is required for my changes)_
- [x] I have made changes in `CHANGELOG.md` file

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -85,7 +83,7 @@ jobs: # Docs: <https://git.io/JvxXE>
 
   lint-changelog:
     name: Lint changelog file
-    runs-on: ubuntu-latest
+    runs-on:  ubuntu-20.04
```
Comment (IgorKorytin) [inline .github/workflows/tests.yml:86]:
```suggestion
    runs-on: ubuntu-20.04
```

File: Makefile [hunk 2]
```diff
@@ -17,19 +17,19 @@ build: ## Build docker images, required for current package environment
 	docker-compose build
 
 latest: clean ## Install latest php dependencies
-	docker-compose run $(RUN_APP_ARGS) app composer update -n --ansi --no-suggest --prefer-dist --prefer-stable
+	docker-compose run $(RUN_APP_ARGS) app composer update -n --ansi --prefer-dist --prefer-stable
 
 install: clean ## Install regular php dependencies
-	docker-compose run $(RUN_APP_ARGS) app composer update -n --prefer-dist --no-interaction --no-suggest
+	docker-compose run $(RUN_APP_ARGS) app composer update -n --prefer-dist --no-interaction
```
Comment (tarampampam) [inline Makefile:23]:
```suggestion
	docker-compose run $(RUN_APP_ARGS) app composer update -n --prefer-dist
```

**Review by tarampampam — CHANGES_REQUESTED** [review]

**Comment by tarampampam** [comment]
:+1: Great thanks for your contribution!

But can you also update:

- [ ] [phpunit.xml](https://github.com/avto-dev/stacked-dumper-laravel/runs/1569407905?check_suite_focus=true#step:11:15) ([example](https://github.com/spiral/roadrunner-laravel/blob/v3.7.0/phpunit.xml.dist))

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -13,15 +13,15 @@ on:
 jobs: # Docs: <https://git.io/JvxXE>
   php:
     name: PHP ${{ matrix.php }}, ${{ matrix.setup }} setup, laravel ${{ matrix.laravel }}
-    runs-on: ubuntu-latest
+    runs-on:  ubuntu-20.04
```
Comment (tarampampam) [inline .github/workflows/tests.yml:16]:
```suggestion
    runs-on: ubuntu-20.04
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog][keepachangelog] and this project adheres to [Semantic Versioning][semver].
 
+## Unreleased
```
Comment (tarampampam) [inline CHANGELOG.md:7]:
```suggestion
## v1.4.0
```

**Review by tarampampam — CHANGES_REQUESTED** [review]

**Pull request merged**